LDAR_Sim/src/weather/daylight_calculator.py

```diff
@@ -31,7 +32,10 @@ def __init__(self, site_lat_lon: tuple, start_date: date, end_date: date):
         lon_ave = site_lat_lon[1]
 
         curr_date = start_date
-
+        # Store the inputs as attributes
```

Comment:
Question: Why do these need to get stored?

Reply:
They were stored to make sure the class could be easily reconstructed without having to manually make the _reconstruct method.